contract-examples/README.md

```diff
@@ -89,8 +83,8 @@ Alternatively, you can copy and paste the `local` network configuration to creat
         "0x86f78c5416151fe3546dece84fda4b4b1e36089f2dbc48496faf3a950f16157c",
         "0x750839e9dbbd2a0910efe40f50b2f3b2f2f59f5580bb4b83bd8c1201cf9a010a"
       ],
-      pollingInterval: "1s"
-    },
+      "pollingInterval": "1s"
```

Comment:
🤔  The hardhat config is usually a `js` file isn't it?

Reply:
TS is officially supported as well